templates/templates/preview/clientconfig_plain_text.template

```diff
@@ -0,0 +1,51 @@
+   
+        Client Configuration:
+            {%- if manifest.dbs %}
+            {%- if manifest.dbs[0].name %}
+            Database identifier: {{ manifest.dbs[0].name }}
+            {%- endif %}
+            {%- endif %}
+            {%- if model.run_settings.colocated_db_settings %}
+            {%- if model.run_settings.colocated_db_settings.db_identifier %}
+            Database identifier: {{ model.run_settings.colocated_db_settings.db_identifier }}
+            {%- endif %}
+            {%- endif %}
+            Database backend: {% set db_type, _ = config.database_cli.split("/")[-1].split("-",1) %}{{ db_type }}
+            {%- if model.run_settings.colocated_db_settings %}
+                {%- if model.run_settings.colocated_db_settings.port %}
+            TCP/IP Port:
+                {{ model.run_settings.colocated_db_settings.port }}
+                {%- endif -%}
+                {%- if model.run_settings.colocated_db_settings.unix_socket %}
+            Socket File: {{ model.run_settings.colocated_db_settings.unix_socket }}
+                {%- endif -%}
+            {%- endif %}
+            {%- if model.run_settings.colocated_db_settings %}
+            Type: Colocated
+            {%- endif %}
+            {%- if manifest.dbs %}
+            {%- if manifest.dbs[0].num_shards > 2 %}
+            Type: Clustered
+            {% else %}
+            Type: Standalone
+            {%- endif %}
+            {%- endif %}
+            {%- if incoming in model.incoming_entities %}
+            Incoming entities (available data sources): 
+            {%- for incoming in model.incoming_entities %}
+                {{ incoming.name }}
+            {%- endfor %}
+            {%- endif %}
+            Outgoing key collision prevention (key prefixing): 
+            {%- if model.query_key_prefixing() == True %}
+                Tensors: On 
+                DataSets: On 
```

Comment:
Could we turn this into a single section without nesting using a custom filter? Does that pattern help us simplify other things... if not, ignore...

```
{% if xxx %}
                Tensors: Off 
                DataSets: Off
                Models/Scripts: Off
                Aggregation Lists: Off
{% else %}
                Tensors: On 
                DataSets: On
                Models/Scripts: On
                Aggregation Lists: On
{% endif %}
```

could become:

```
                Tensors: {% model.query_key_prefixing | as_toggle %} 
                DataSets: {% model.query_key_prefixing | as_toggle %} 
                Models/Scripts: {% model.query_key_prefixing | as_toggle %} 
                Aggregation Lists: {% model.query_key_prefixing | as_toggle %} 
```

with a [custom filter](https://jinja.palletsprojects.com/en/3.1.x/api/#custom-filters) in the vein of:

```py
@pass_eval_context
def as_toggle(eval_ctx, value: bool):
  return "On" if value else "Off"

```

Reply:
implemented a custom filter